components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2613,27 +2589,6 @@ impl ApplyFsm {
 
         self.delegate.metrics = ApplyMetrics::default();
         self.delegate.term = apply.term;
-        let prev_state = (
```

Comment:
The check can be reserved. And you only need to set commit index of `apply_state` using `entries.last().unwrap().get_index()`.

Reply:
Done.